### PR TITLE
Update get-help-support.md

### PR DESCRIPTION
**Pull request by msbemba**
Checked two tenants and it shows option to contact partner with phone and email but no option to directly open a ticket with Microsoft.

Per issue #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9906

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/get-help-support.md [hunk 1]
```diff
@@ -57,8 +57,7 @@ Save time by starting your service request online. We'll help you find a solutio
 5. Enter a description of your issue, confirm your contact number and email address, select your preferred contact method, and then select **Contact me**. The expected wait time is indicated in the **Contact support** pane.
 
     > [!NOTE]
-    > If you bought your subscription through a partner, you first see the contact information for that partner. Alternatively, select **New Microsoft service request** at the bottom of the pane.
-
+    > If you bought your subscription through a partner, you first see the contact information for that partner.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-support.md:60]:
```suggestion
    > If you bought your subscription through a partner, you first see the contact information for that partner.
    
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder

**Comment by cmcatee-MSFT** [comment]
Change has already been incorporated into the article.

**Comment by msbemba** [comment]
@cmcatee-MSFT  The change is If you bought your subscription through a partner, you first see the contact information for that partner and there is no alternate option to contact Microsoft.

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder

**Comment by cmcatee-MSFT** [comment]
The note was removed from the article, and the comment about getting support through a partner was moved to the Important note at the start of the Online support section. No change is needed.

**Pull request closed without merging**